app/views/orchestration_stack/show.html.haml

```diff
@@ -6,7 +6,7 @@
     = render(:partial => "layouts/item")
   - else
     - if %w(instances security_groups children).include?(@display)
-      = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@orchestration_stack.id}"}
+      = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
```

Comment:
as we talked we miss the helper method for that issue right now, this building of the ```:action_url``` is used across the lot of views